src/jira/util/jwt.ts

```diff
@@ -28,18 +29,22 @@ export enum TokenType {
 	context = "context"
 }
 
-export function extractJwtFromRequest(req: Request): string | undefined {
-	const tokenInQuery = req.query?.[JWT_PARAM];
+let secureJiraHostInCookies;
+
+
+function extractJwtFromRequest(req: Request): string | undefined {
 
-	// JWT appears in both parameter and body will result query hash being invalid.
+	booleanFlag(BooleanFlags.SECURE_JIRAHOST_IN_COOKIES, false)
```

Comment:
please use `await` instead

Reply:
We don't want to change the whole chain to be async @mboudreau cause it is pretty long.